client/python/README.md

```diff
@@ -1,19 +1,26 @@
-# Armada Python client
+Armada Python Client
+=
+<hr />
 
+## How To Use This Application
+***
+
+> **Do Not Fork** this application is meant to be used from **[armada_client](https://pypi.org/project/armada-client/)**
```

Comment:
Alternatively, this line could be completely removed.

```suggestion
These instructions are intended for end-users. If you wish to develop against armada_client, please see our [documentation on contributing](CONTRIBUTING.md)
```

(and obviously if you took this suggestion, you'd remove the contributing link below.

Reply:
@kindyluv doesn't look like this comment has been addressed